### ISSUE TITLE
Create User-Facing Guidance Documentation for Senders Using CSV Uploader - Instructional Design

### ISSUE THREAD

**Issue by loripusey** [CDCgov/prime-reportstream#3455]
## Problem Statement
As a Sender, I need to prepare and upload my CSV file to adhere to ReportStream standard file requirements.

## Considerations
- Develop content only (how to format and offer the info will be in a different story)
- Meta data for the schema such as: (comes from engineering)
    - Fields & their formats
    - Required fields
- CSV file template that considers the standard scheme (comes from engineering)
- Other instructions (e.g. how to look up test kit info on CDC's site) (comes from engineering)
- Self-guided instructions because we may not offer much direct or 1/1 support

This is the document outlining guidance for Experience team on how the new standard schema will be based: https://cdc.sharepoint.com/:w:/r/teams/USDSatCDC/Shared%20Documents/PRIME%20ReportStre[…]cx?d=w0bee04468e4f4736bd604831fb1d75ec&csf=1&web=1&e=lZ6KXS

## To Do / Deliverables

- [x] Instructional Design Document outlining learning content for users who will be sending test results data reporting to ReportStream via CSV uploader

**Comment by BerniXiongA6** [comment]
Updates: 

- [Here's the current DRAFT](https://cdc.sharepoint.com/:w:/r/teams/USDSatCDC/Shared%20Documents/PRIME%20ReportStream/Team%20Documents/Experience%20Team/Instructional%20Design/DRAFT%20-%20Instructional%20Design%20Doc%20-%20Preparing%20CSV%20to%20comply%20with%20COVID-19%20Standard%20Schema%20-%20Rev%202021.12.20.docx?d=w9d0556ceb58b49418e25e4d13e3777a8&csf=1&web=1&e=hYVOHg) of the instruction design document outlining the CSV Guide. It covers the structure of the guide, the potential topics, some preliminary content, and other items to discuss and consider. Once the schema is completed and some of the pending questions inside the outline are answered and we decide the delivery method for the guide, I can add more instructional and design content. 

- @rhood23699 and I are connecting on Tuesday AM 12/21/2021 to review the doc and discuss the standard schema requirements

**Comment by BerniXiongA6** [comment]
12.21.2021 Met with @rhood23699 and [reviewed documentation outlining the new standard schema and guidance/notes](https://cdc.sharepoint.com/:w:/r/teams/USDSatCDC/Shared%20Documents/PRIME%20ReportStre[…]cx?d=w0bee04468e4f4736bd604831fb1d75ec&csf=1&web=1&e=pqRXLk) for Experience team to consider. Today I'm prioritizing updating the instruction design outline to include guidance discussed this morning. Waiting on a CSV sample template from Rick Hood to be shared with me by EOD today.

**Comment by BerniXiongA6** [comment]
This ticket is done. Creating new ticket (will return here with #) to prioritize the MVP with the actual content for Sprint 32.
